api/lib/infrastructure/repositories/campaign-participant-repository.js

```diff
@@ -140,6 +141,7 @@ async function _getOrganizationLearner(campaignId, userId, domainTransaction) {
       function () {
         this.on('campaign-participations.organizationLearnerId', 'organization-learners.id');
         this.on('campaign-participations.campaignId', 'campaigns.id');
+        this.on('campaign-participations.deletedAt', knex.raw('IS'), knex.raw('NULL'));
```

Comment:
`this.on('campaign-participations.deletedAt', null)` ne fonctionne pas ?

Reply:
Ah pas penser à tester 💡

Reply:
marche pas 😭  ça aurait été cool !